projects/elife.yaml

```diff
@@ -372,6 +372,8 @@ master-server:
     vagrant:
         # 16.04, deprecated
         box: bento/ubuntu-16.04
+        ports:
+            8200: 8200 # Vault GUI
```

Comment:
yes, it's already open on AWS
Though it's not the UI port, it's just Vault's port hosting both a REST API and the `/ui`.

Reply:
Don't quite follow - it's not the UI port but it hosts `/ui` ?

Reply:
It hosts both `/api` and `/ui` (not sure if the first path is correct, but you get the point)